examples/company.rb

```diff
@@ -1,4 +1,4 @@
 require 'clearbit'
 require 'pp'
 
-pp Clearbit::Company.find(domain: 'stripe.com')
+pp Clearbit::Enrichment::Company.find(domain: 'stripe.com', stream: true, blah: 1)
```

Comment:
Can we make the blah: 1 something that actually does something, or otherwise remove it?